test/fixtures/ExtraPage.md

```diff
@@ -0,0 +1,15 @@
+# Extra Page Title
```

Comment:
I tried that, but it broke some of the existing tests.  It seems that if we add a `#` to the README, then we can't test the case where there is no `#` in the file, no `title` in the config, and it falls back to the original behavior of using the filename as the menu title.